src/workloads/query/TimeSeriesSortCompound.yml

```diff
@@ -88,8 +88,7 @@ Actors:
           {$skip: 1e10},
         ]
         cursor: {}
-        allowDiskUse: true
-
+        hint: {m: 1, t: 1}
```

Comment:
And we don't need to use disk for bounded sort?

Reply:
It's possible to make bounded sort spill, but this test won't do it. BoundedSort is like a sorted merge, so the memory it uses is minimal, correlated with how many buckets it needs to have open. With the way the buckets are laid out I don't think it has enough buckets open to spill.